### PR TITLE
Change library name

### PR DESCRIPTION
**Pull request by fedefernandez**
Big PR but only the name of files and packages has been changed.

Please @juanpedromoreno could you take a look? Thanks

### CODE REVIEW

File: src/test/scala/com/fortysevendeg/mvessel/DatabaseSpec.scala [hunk 1]
```diff
@@ -20,14 +20,14 @@ trait SQLDroidDatabaseSpecification
...
-  trait SQLDroidDatabaseScope
+  trait DatabaseScope
     extends Scope {
 
     val mockedDatabase = mock[SQLiteDatabase]
 
     val cursor = mock[Cursor]
 
-    val sqlDroid = new SQLDroidDatabase(databaseName, timeout, retry, flags) {
+    val databse = new Database(databaseName, timeout, retry, flags) {
```
Comment (juanpedromoreno) [inline src/test/scala/com/fortysevendeg/mvessel/DatabaseSpec.scala]: Maybe you misspelled this val. should be `database`?

File: src/test/scala/com/fortysevendeg/mvessel/DatabaseSpec.scala [hunk 2]
```diff
@@ -35,25 +35,25 @@ trait SQLDroidDatabaseSpecification
...
-class SQLDroidDatabaseSpec extends SQLDroidDatabaseSpecification {
+class DatabaseSpec extends DatabaseSpecification {
 
   "rawQuery" should {
 
-    "returns the Cursor when the method rawQuery in SQLiteDatabase works" in new SQLDroidDatabaseScope {
+    "returns the Cursor when the method rawQuery in SQLiteDatabase works" in new DatabaseScope {
       mockedDatabase.rawQuery(any, any) returns cursor
-      sqlDroid.rawQuery(sql = sql, selectionArgs = Array()) shouldEqual cursor
+      databse.rawQuery(sql = sql, selectionArgs = Array()) shouldEqual cursor
```
Comment (juanpedromoreno) [inline src/test/scala/com/fortysevendeg/mvessel/DatabaseSpec.scala]: Same here. Maybe you misspelled this val. should be `database`?

File: build.sbt [hunk 3]
```diff
@@ -47,5 +47,5 @@ lazy val root = (project in file(".")).
   enablePlugins(BuildInfoPlugin).
   settings(
     buildInfoKeys := Seq[BuildInfoKey](name, version, scalaVersion, sbtVersion),
-    buildInfoPackage := "com.fortysevendeg.android.sqlite"
+    buildInfoPackage := "com.fortysevendeg.mvessel"
```
Comment (juanpedromoreno) [inline build.sbt:50]: You should change the sbt name to `mvessel` as well.

https://github.com/47deg/mvessel/blob/fede-refactor-name/build.sbt#L11

**Comment by juanpedromoreno** [comment]
LGTM!

File: src/main/scala/com/fortysevendeg/mvessel/statement/Statement.scala [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
+import com.fortysevendeg.mvessel.resultset.ResultSet
+import com.fortysevendeg.mvessel.statement.StatementInfo._
+import com.fortysevendeg.mvessel.{Connection, Database, WrapperNotSupported, javaNull}
 
 import scala.collection.mutable
 import scala.util.{Failure, Success, Try}
 
-class SQLDroidStatement(
-  sqlDroidConnection: SQLDroidConnection,
+class Statement(
```
Comment (47degci) [inline src/main/scala/com/fortysevendeg/mvessel/statement/Statement.scala:14]: Codacy Issue found: [Number of methods in class exceeds 30](https://www.codacy.com/app/47deg/scala-sqlite-droid/file/822213828/issues/source?bid=2442814&fileBranchId=2752966#l14)

File: src/main/scala/com/fortysevendeg/mvessel/statement/Statement.scala [hunk 5]
```diff
@@ -30,11 +30,11 @@ class SQLDroidStatement(
...
 
-  protected var connection: Option[SQLDroidConnection] = Option(sqlDroidConnection)
+  protected var connection: Option[Connection] = Option(sqlConnection)
 
   protected var maxRows: Option[Int] = None
 
-  protected var resultSet: Option[ResultSet] = None
+  protected var resultSet: Option[SQLResultSet] = None
 
   protected var updateCount: Option[Int] = None
```
Comment (47degci) [inline src/main/scala/com/fortysevendeg/mvessel/statement/Statement.scala:39]: Codacy Issue found: [Avoid mutable fields](https://www.codacy.com/app/47deg/scala-sqlite-droid/file/822213828/issues/source?bid=2442814&fileBranchId=2752966#l39)

File: src/main/scala/com/fortysevendeg/mvessel/metadata/DatabaseMetaData.scala [hunk 6]
```diff
@@ -58,23 +58,23 @@ class SQLDroidDatabaseMetaData(connection: Connection)
...
-  override def getCatalogs: ResultSet = catalogsStatement.executeQuery()
+  override def getCatalogs: SQLResultSet = catalogsStatement.executeQuery()
 
   override def getColumnPrivileges(
     catalog: String,
     schema: String,
     table: String,
     columnNamePattern: String
-    ): ResultSet = columnPrivilegesStatement.executeQuery()
+    ): SQLResultSet = columnPrivilegesStatement.executeQuery()
```
Comment (47degci) [inline src/main/scala/com/fortysevendeg/mvessel/metadata/DatabaseMetaData.scala:71]: Codacy Issue found: [Method is longer than 50 lines](https://www.codacy.com/app/47deg/scala-sqlite-droid/file/822213819/issues/source?bid=2442814&fileBranchId=2752966#l72)

File: src/main/scala/com/fortysevendeg/mvessel/Connection.scala [hunk 7]
```diff
@@ -39,7 +40,7 @@ class SQLDroidConnection(
 
   val alreadyClosedErrorMessage = "Database connection closed"
 
-  private[this] var sqliteDatabase: Option[SQLDroidDatabase] = createDatabase()
+  private[this] var sqliteDatabase: Option[Database] = createDatabase()
```
Comment (47degci) [inline src/main/scala/com/fortysevendeg/mvessel/Connection.scala:44]: Codacy Issue found: [Avoid mutable fields](https://www.codacy.com/app/47deg/scala-sqlite-droid/file/822213811/issues/source?bid=2442814&fileBranchId=2752966#l45)

File: src/main/scala/com/fortysevendeg/mvessel/statement/Statement.scala [hunk 8]
```diff
@@ -30,11 +30,11 @@ class SQLDroidStatement(
 
   protected val batchList = new mutable.MutableList[String]
 
-  protected var connection: Option[SQLDroidConnection] = Option(sqlDroidConnection)
+  protected var connection: Option[Connection] = Option(sqlConnection)
```
Comment (47degci) [inline src/main/scala/com/fortysevendeg/mvessel/statement/Statement.scala:33]: Codacy Issue found: [Avoid mutable fields](https://www.codacy.com/app/47deg/scala-sqlite-droid/file/822213828/issues/source?bid=2442814&fileBranchId=2752966#l33)

File: src/main/scala/com/fortysevendeg/mvessel/package.scala [hunk 9]
```diff
@@ -1,6 +1,6 @@
-package com.fortysevendeg.android
+package com.fortysevendeg
 
-package object sqlite {
+package object mvessel {
 
   val javaNull = null
```
Comment (47degci) [inline src/main/scala/com/fortysevendeg/mvessel/package.scala:5]: Codacy Issue found: [Avoid using null](https://www.codacy.com/app/47deg/scala-sqlite-droid/file/822213823/issues/source?bid=2442814&fileBranchId=2752966#l5)

File: src/main/scala/com/fortysevendeg/mvessel/statement/Statement.scala [hunk 10]
```diff
@@ -30,11 +30,11 @@ class SQLDroidStatement(
...
 
   protected var maxRows: Option[Int] = None
 
-  protected var resultSet: Option[ResultSet] = None
+  protected var resultSet: Option[SQLResultSet] = None
```
Comment (47degci) [inline src/main/scala/com/fortysevendeg/mvessel/statement/Statement.scala:37]: Codacy Issue found: [Avoid mutable fields](https://www.codacy.com/app/47deg/scala-sqlite-droid/file/822213828/issues/source?bid=2442814&fileBranchId=2752966#l37)

File: src/main/scala/com/fortysevendeg/mvessel/statement/Statement.scala [hunk 11]
```diff
@@ -30,11 +30,11 @@ class SQLDroidStatement(
 
   protected val batchList = new mutable.MutableList[String]
 
-  protected var connection: Option[SQLDroidConnection] = Option(sqlDroidConnection)
+  protected var connection: Option[Connection] = Option(sqlConnection)
 
   protected var maxRows: Option[Int] = None
```
Comment (47degci) [inline src/main/scala/com/fortysevendeg/mvessel/statement/Statement.scala:35]: Codacy Issue found: [Avoid mutable fields](https://www.codacy.com/app/47deg/scala-sqlite-droid/file/822213828/issues/source?bid=2442814&fileBranchId=2752966#l35)

**Pull request merged**